tests/playwright-test/test-ignore.spec.ts

```diff
@@ -390,3 +390,28 @@ test('should only match files with JS/TS file extensions', async ({ runInlineTes
   expect(result.exitCode).toBe(0);
   expect(result.passed).toBe(2);
 });
+
+test('should work against a source mapped file', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'out/foo.spec.js': `"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+//@no-header
+const test_1 = require(${JSON.stringify(path.join(__dirname, 'entry'))});
+test_1.default('passes', async () => {
+});
+//# sourceMappingURL=foo.spec.js.map`,
+
+    'out/foo.spec.js.map': `{"version":3,"file":"foo.spec.js","sourceRoot":"","sources":["../src/foo.spec.ts"],"names":[],"mappings":";;AAAA,YAAY;AACZ,2CAAoC;AAEpC,cAAI,CAAC,QAAQ,EAAE,KAAK,IAAG,EAAE;AACzB,CAAC,CAAC,CAAC"}`,
+
+    'src/foo.test.ts': `//@no-header
+import test from '@playwright/test';
+
+test('passes', async() => {
+});
+`
+  }, {
+    args: ['-c', 'out', 'foo.spec.ts']
+  });
+  expect(result.exitCode).toBe(0);
```

Comment:
The important part for the VSCode extension is that even if you have JS as input. it should use as output file names the TS file names.

Reply:
Where's the code that does this? Error messages will appear source mapped, but the suite titles will show the requireFile.

Reply:
Its not about error messages, we have two use cases which we need to solve:
a) find all tests via the test explorer. Its basically `npx playwright test out/` there we currently get in the JSON report the JS files, because in the out/ dir are only JS files. But to reference them correctly in vsc I need the TS references with line numbers.
b) the user opens a file, e.g. foo.test.ts With the current setup it won't work at all, because babel-typescript can't compile it, so its broken then.